src/core/audio/AudioDevice.cpp

```diff
@@ -130,17 +130,19 @@ void AudioDevice::renamePort( AudioPort * )
 int AudioDevice::convertToS16(const SampleFrame* _ab,
 								const fpp_t _frames,
 								int_sample_t * _output_buffer,
-								const bool _convert_endian )
+
+								const bool _convert_endian)
 {
+	const unsigned int channels = 2;
 	if( _convert_endian )
 	{
 		for( fpp_t frame = 0; frame < _frames; ++frame )
 		{
-			for( ch_cnt_t chnl = 0; chnl < channels(); ++chnl )
+			for (ch_cnt_t chnl = 0; chnl < channels; ++chnl)
 			{
 				auto temp = static_cast<int_sample_t>(AudioEngine::clip(_ab[frame][chnl]) * OUTPUT_SAMPLE_MULTIPLIER);
 
-				( _output_buffer + frame * channels() )[chnl] =
+				( _output_buffer + frame * channels )[chnl] =
```

Comment:
```suggestion
				(_output_buffer + frame * channels)[chnl] =
```

Reply:
Btw what's this used for?

Reply:
I don't know.

But audio exporting classes use it.